war/src/main/js/widgets/config/tabbar.less

```diff
@@ -151,156 +166,83 @@
         }
       }
 
-      .section-header{
-        margin: 30px 0px 5px -10px;
-        font-size: 1.25rem;
-        padding: 10px 15px;
-        border: none;
-        border-top: 1px solid #ddd;
-      }
-      .attach-previous {
-        margin-left:3px;
-      }
-      .validation-error-area td{
-        padding:0 5px;
+      // Help & descriptions
+      .help {
+        background: var(--configure-job-help-area-bg-color);
       }
 
-      .stapler-class-bag td {
-        padding: 0px;
-      }
-      
-      td.setting-name{
-        line-height:29px;
-      }
-      td.setting-main{
-        input[type='radio'],
-        input[type='checkbox']{
-          position:relative;
-          top:.33em;
-        }
+      p,
+      .help,
+      .setting-description {
+        font-size: var(--font-size-base);
       }
-      td{
-        padding:5px;
```

Comment:
we need to keep this rule, otherwise things will get too packed

Reply:
Really open  for suggestions though

Reply:
isn't that moved to line 186?

Reply:
done